.github/workflows/quarto-book-netlify.yaml

```diff
@@ -0,0 +1,55 @@
+on:
+  push:
+    branches: main
+  pull_request:
+    branches: main
+  # to be able to trigger a manual build
+  workflow_dispatch:
+  schedule:
+    # run every day at 11 PM
+    - cron: '0 23 * * *'
+
+name: Render and deploy Book to Netlify
+
+env:
+  isExtPR: ${{ github.event.pull_request.head.repo.fork == true }}
+
+jobs:
+  build-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install Quarto
+        uses: quarto-dev/quarto-actions/install-quarto@v1
+        with:
+          # To install LaTeX to build PDF book
+          tinytex: true
+          # uncomment below and fill to pin a version
+          # version: 0.9.105
+
+      # Install any computing dependency you may need, usually either python and Jupyter or R
```

Comment:
Don't we need some pak invocation here to install all the needed packages?

Reply:
Yeah, I think so. I've done Quarto / GitHub action with renv but not with a DESCRIPTION file and pak. I also couldn't get the action to run on the PR to verify ☹️